javascript/javascript1/week4/voice-assistant.js

```diff
@@ -0,0 +1,44 @@
+let userName = [];
+let myTodo = [];
+function getReply(command) {
+  if (command.includes("my name is")) {
+    let commandStr = command.split(" ");
```

Comment:
It's better to use full words for variable names. It makes it faster to read the code.